### DISCUSSION TITLE
代理成功，但IP地址却在国内，而且省份未变，请问如何解决？

### DISCUSSION THREAD

**Discussion by Flyingpen** [vvbbnn00/WARP-Clash-API#185] [Q&A]
首先感谢作者开发了这个神奇的程序！感谢！
然后说说我的问题——
1、代理成功后，能正常访问google、TG等，但google一下IP，却显示在国内；
2、因为是国内IP，所以一些IP封锁的网站无法访问。比如我常用的OPENAI，GEMINI，国外版COZE等；
3、部署用的是render免费实例，服务器地址在新加坡。

请问这种情况应该如何处理？

**Comment by vvbbnn00** [comment]
你好，这和WARP的原理有关，可以参照：https://github.com/vvbbnn00/WARP-Clash-API/issues/65
要想解决这个问题，您可能需要用到链式代理等类似的方式：https://wiki.metacubex.one/config/proxies/dialer-proxy/
